src/components/grid-aware/OneParagraphBlock/OneParagraphBlock.tsx

```diff
@@ -1,11 +1,22 @@
-import PropTypes from "prop-types";
-import React from "react";
+import * as React from "react";
 
-import Button from "../../inline/Button";
+import Button, { ButtonProps } from "../../inline/Button";
 
 import s from "./OneParagraphBlock.module.css";
 
-const OneParagraphBlock = ({ title, descriptions, ctaButtons }) => {
+/* Subcomponents */
+
+type OneParagraphBlockProps = {
+  title: string;
+  descriptions: string;
```

Comment:
This doesn't match the original PropTypes type that was assigned to it, which was `PropTypes.node`. The PropTypes version is the correct type, because in the actual page where we instantiate this component, we're passing in a React fragment, not a string:

https://github.com/ShelterTechSF/sheltertech.org/blob/780ef84c827225633c8a1dc975faf4dc4f7eeae3/src/pages/volunteer/index.js#L150-L179

This isn't triggering a TypeScript error because we haven't converted the page components yet, but it will once we do that. Could you replace this with `React.ReactNode`, which is the equivalent to `PropTypes.node`?

Reply:
Changed description prop type to React.ReactNode.